app/src/main/java/ar/com/wolox/android/example/ui/training/login/LoginActivity.java

```diff
@@ -0,0 +1,20 @@
+package ar.com.wolox.android.example.ui.training.login;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+
+/**
```

Comment:
This is a JavaDoc comment (but empty). Sometimes it would be good to document, but if you don't do it, please don't leave it empty.